stdlog/stdlog-intern.h

```diff
@@ -28,6 +28,14 @@
 #include <sys/un.h>
 #include "stdlog.h"
 
+#ifndef O_CLOEXEC
```

Comment:
i don't know about usage on other platforms, but it's also a common fallback for older Linux systems that don't have it

yes, adding fallback to fcntl would be a more complete solution, but i figured it wasn't worth the effort.  if someone wants to, they could make that change easily on top of my change here, so we at least get incremental improvements.